indica/datatypes.py

```diff
@@ -119,6 +120,7 @@
         "Centrifugal asymmetry multiplier",
         "none",
     ),
+    "faraday_rotation_integrated": ("rad", "none"),
```

Comment:
The first item of the tuple is the DataArray LongName and the second is the Unit string identifier.
You'll have to add the Unit (e.g. "angle_radians" : "rad"), and you can pick a sensible LongName (e.g. "Faraday rotation").